### ISSUE TITLE
InternalError: Internal error { code: -32603 } on createSellOrder()

### ISSUE THREAD

**Issue by mayavolt** [ProjectOpenSea/opensea-js#396]
After updating OpenSea-js to version 2.0.1 and Nodejs to 16 (also trying 8-15) I started getting the error:
`InternalError: Internal error { code: -32603 }`
This happens after calling the `createSellOrder()` function. 

Part of my code:
```
async function sellItem(item) {
    try {
        const dutchAuctionSellOrder = await seaport.createSellOrder({
            asset: {
                tokenId: item.id,
                tokenAddress: NFT_CONTRACT_ADDRESS,
                schemaName: WyvernSchemaName.ERC1155,
            },
            startAmount: item.startAmount,
            endAmount: item.endAmount,
            expirationTime: Math.floor(Date.now() / 1000) + 240,
            accountAddress: OWNER_ADDRESS,
        });
    } catch (e) {
        console.log(e) // output--> InternalError: Internal error { code: -32603 }
    }
}
```
This used to work great before update.

**Comment by Toeplitz** [comment]
I have problem with this method as well after the upgrade, but it just hangs silently. Same versions.

**Comment by d-s-i** [comment]
Any idea why it does that ? I have the same error with ceateBuyOrder
```
            const offer = await seaport.createBuyOrder({
              asset: {
                tokenId: id,
                tokenAddress: addresses.contract,
                schemaName : WyvernSchemaName.ERC721 // WyvernSchemaName. If omitted, defaults to 'ERC721'. Other options include 'ERC20' and 'ERC1155'
              },
              accountAddress: addresses.bidder,
              // Value of the offer, in units of the payment token (or wrapped ETH if none is specified):
              startAmount: bidArgs.readableBidAmount,
              expirationTime: bidArgs.expirationTime
            });
```

**Comment by jackwolt** [comment]
> Any idea why it does that ? I have the same error with ceateBuyOrder

Still getting this error. I don't understand what the problem is. Did you manage to find a solution?

**Comment by esteban-OpenSea** [comment]
Wanted to follow up on this since I haven't seen any activity with this recently. Is this still an issue? Have you upgraded to the most recent versions?

**Comment by avm70** [comment]
> Wanted to follow up on this since I haven't seen any activity with this recently. Is this still an issue? Have you upgraded to the most recent versions?

It is still an issue. Failed to create sell order.
`[InternalError: Internal error] { code: -32603 }
Failed: Error: You declined to authorize your auction
    at OpenSeaPort.<anonymous> (/home/avm/opensea/test/node_modules/opensea-js/src/seaport.ts:888:13)
    at step (/home/avm/opensea/test/node_modules/opensea-js/lib/seaport.js:63:23)
    at Object.throw (/home/avm/opensea/test/node_modules/opensea-js/lib/seaport.js:44:53)
    at rejected (/home/avm/opensea/test/node_modules/opensea-js/lib/seaport.js:36:65)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)`

**Comment by esteban-OpenSea** [comment]
Is there a reason why you're using v2 instead of v3 of the sdk? Are you running into issues installing the latest version of the SDK and Node?

**Comment by avm70** [comment]
I used node 8.11.2 before sdk update.
After sdk update I installed new version of sdk. This error occured after update.
Then I updated node to 16.14.2
Sdk not installing with err
`npm ERR! code 127
npm ERR! git dep preparation failed
npm ERR! command /home/avm/.nvm/versions/node/v16.14.2/bin/node /home/avm/.nvm/versions/node/v16.14.2/lib/node_modules/npm/bin/npm-cli.js install --force --cache=/home/avm/.npm --prefer-offline=false --prefer-online=false --offline=false --no-progress --no-save --no-audit --include=dev --include=peer --include=optional --no-package-lock-only --no-dry-run
npm ERR! > wyvern-schemas@0.14.1 prepublish
npm ERR! > yarn build
npm ERR! npm WARN using --force Recommended protections disabled.
npm ERR! npm WARN deprecated safe-event-emitter@1.0.1: Renamed to @metamask/safe-event-emitter
npm ERR! npm WARN deprecated mkdirp-promise@5.0.1: This package is broken and no longer maintained. 'mkdirp' itself supports promises now, please switch to that.
npm ERR! npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm ERR! npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm ERR! npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm ERR! npm WARN deprecated eth-sig-util@1.4.2: Deprecated in favor of '@metamask/eth-sig-util'
npm ERR! npm WARN deprecated ethereumjs-account@3.0.0: Please use Util.Account class found on package ethereumjs-util@^7.0.6 https://github.com/ethereumjs/ethereumjs-util/releases/tag/v7.0.6
npm ERR! npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm ERR! npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
npm ERR! npm WARN deprecated ethereumjs-block@2.2.2: New package name format for new versions: @ethereumjs/block. Please update.
npm ERR! npm WARN deprecated ethereumjs-block@2.2.2: New package name format for new versions: @ethereumjs/block. Please update.
npm ERR! npm WARN deprecated ethereumjs-block@2.2.2: New package name format for new versions: @ethereumjs/block. Please update.
npm ERR! npm WARN deprecated nomnom@1.8.1: Package no longer supported. Contact support@npmjs.com for more info.
npm ERR! npm WARN deprecated ethereumjs-tx@2.1.2: New package name format for new versions: @ethereumjs/tx. Please update.
npm ERR! npm WARN deprecated ethereumjs-tx@2.1.2: New package name format for new versions: @ethereumjs/tx. Please update.
npm ERR! npm WARN deprecated ethereumjs-tx@2.1.2: New package name format for new versions: @ethereumjs/tx. Please update.
npm ERR! npm WARN deprecated uuid@2.0.1: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm ERR! npm WARN deprecated ethashjs@0.0.8: New package name format for new versions: @ethereumjs/ethash. Please update.
npm ERR! npm WARN deprecated multicodec@1.0.4: This module has been superseded by the multiformats module
npm ERR! npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm ERR! npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm ERR! npm WARN deprecated uuid@3.2.1: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm ERR! npm WARN deprecated uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm ERR! npm WARN deprecated mkdirp@0.5.1: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
npm ERR! npm WARN deprecated ethereumjs-blockchain@4.0.4: New package name format for new versions: @ethereumjs/blockchain. Please update.
npm ERR! npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm ERR! npm WARN deprecated multibase@0.6.1: This module has been superseded by the multiformats module
npm ERR! npm WARN deprecated multibase@0.7.0: This module has been superseded by the multiformats module
npm ERR! npm WARN deprecated ethereumjs-tx@1.3.7: New package name format for new versions: @ethereumjs/tx. Please update.
npm ERR! npm WARN deprecated multicodec@0.5.7: This module has been superseded by the multiformats module
npm ERR! npm WARN deprecated ethereumjs-vm@2.6.0: New package name format for new versions: @ethereumjs/vm. Please update.
npm ERR! npm WARN deprecated axios@0.17.1: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410
npm ERR! npm WARN deprecated ethereumjs-block@1.7.1: New package name format for new versions: @ethereumjs/block. Please update.
npm ERR! npm WARN deprecated ethereumjs-common@1.5.2: New package name format for new versions: @ethereumjs/common. Please update.
npm ERR! npm WARN deprecated popper.js@1.14.3: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1
npm ERR! npm WARN deprecated ethereumjs-vm@4.2.0: New package name format for new versions: @ethereumjs/vm. Please update.
npm ERR! npm WARN deprecated cids@0.7.5: This module has been superseded by the multiformats module
npm ERR! npm WARN deprecated core-js@2.6.12: core-js@<3.4 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.
npm ERR! sh: 1: yarn: not found
npm ERR! npm ERR! code 127
npm ERR! npm ERR! path /home/avm/.npm/_cacache/tmp/git-clonerwzFgJ
npm ERR! npm ERR! command failed
npm ERR! npm ERR! command sh -c yarn build
npm ERR! 
npm ERR! npm ERR! A complete log of this run can be found in:
npm ERR! npm ERR!     /home/avm/.npm/_logs/2022-04-20T17_34_41_022Z-debug-0.log

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/avm/.npm/_logs/2022-04-20T17_34_38_720Z-debug-0.log`

**Comment by avm70** [comment]
> Is there a reason why you're using v2 instead of v3 of the sdk? Are you running into issues installing the latest version of the SDK and Node?

I installed opensea-js with yarn still got this
`[InternalError: Internal error] { code: -32603 }
Failed: Error: You declined to authorize your auction
    at OpenSeaPort.<anonymous> (/home/avm/opensea/test_12/node_modules/opensea-js/src/seaport.ts:888:13)
    at step (/home/avm/opensea/test_12/node_modules/opensea-js/lib/seaport.js:63:23)
    at Object.throw (/home/avm/opensea/test_12/node_modules/opensea-js/lib/seaport.js:44:53)
    at rejected (/home/avm/opensea/test_12/node_modules/opensea-js/lib/seaport.js:36:65)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)`
